### DISCUSSION TITLE
[2024-09-23] GUI Team Demo Day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#11154] [Weekly Demo Day]
* #11151 

[Screencast from 2024-09-23 13-19-31.webm](https://github.com/user-attachments/assets/aa28708e-f4c9-400a-966d-1d82abe99f1e)

* #11108

 [Screencast from 2024-09-17 13-20-40.webm](https://github.com/user-attachments/assets/561adebb-7d93-4939-9dde-5453d5f304be)

* Fixes for release:
  * #11014
  * #11001 - this is, `enso://` paths.
  * #11030 - The feature still does not work 100%, because icon are not updated for `*.enso-project` files, and user is asked when running first time (but Enso app is a prominent option).
  * #11057
  * #11091 - an issue reappear, where picking numeric widget in drop-down disallowed any other option in this drop-down.
  * #11099
  * #11103

**Comment by vitvakatu** [comment]
A bunch of small stuff before the release:
- https://github.com/enso-org/enso/pull/10953
- https://github.com/enso-org/enso/pull/11005
- https://github.com/enso-org/enso/pull/11031

Working on input nodes (https://github.com/enso-org/enso/issues/10696)

https://github.com/user-attachments/assets/1764dc27-713f-4057-ba57-f42c1733318b

**Comment by kazcw** [comment]
- #11077
  - New API is simpler, safer, ensures consistency
  - Fixes:
    - Fullscreen mode: Fix rendering size of scatter plot and other visualizations
    - JSON visualization interactivity: Fix intermittent incorrectly non-interactive state
    - #10809
    - Viz toolbars: Fix squished-looking rightmost button.
- #10666 -- 4 little issues
- #11144 -- 2 little issues
- #11016 -- prerequisite for new viz API, also allowed some cleanup

**Comment by Frizi** [comment]
- Added support for touch-based gestures for graph editor pan/zoom, drag and selection -  https://github.com/enso-org/enso/pull/11056
- Fixed empty-graph state handling, deleting last node and creating first node - https://github.com/enso-org/enso/pull/10990
- Added support for multiple-file drag-and-drop upload - https://github.com/enso-org/enso/pull/10979

https://github.com/user-attachments/assets/00b2279d-2acf-468b-8c3c-aa6885cba23d

- Investigated visual text clipping issue - looks like a font metadata error https://github.com/enso-org/enso/issues/10692

**Comment by hubertp** [comment]
Relevant backend changes:
- disable signing of artifacts for Mac on PRs and develop #11094
- wasm-pack upgrade/sync #11078 
- single node execution #11075 (ongoing work)
